pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -677,7 +678,7 @@ var _ = Describe("Manager", func() {
 			xmlDomain, err := xml.MarshalIndent(domainSpec, "", "\t")
 			Expect(err).ToNot(HaveOccurred())
 			checkIfDiskReadyToUse = func(filename string) (bool, error) {
-				Expect(filename).To(Equal("/var/run/kubevirt/hotplug-disks/hpvolume1.img"))
+				Expect(filename).To(Equal(filepath.Join(baseHotplugDiskDir, "/hpvolume1.img")))
```

Comment:
I think there are some other places that need this change, for instance: https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/converter/converter_test.go#L3128.  Double check you have all the places...

Reply:
I changed it in all places in `virtwrap` package. Changing this in other places would require us to have this const declared in a more global place like v1 schema like you suggested in https://github.com/kubevirt/kubevirt/pull/8355#discussion_r954238551. 

Please let me know if you would like me to do that.